bbb-webhooks/README.md

```diff
@@ -1,97 +1,72 @@
 
-You can read the full documentation at: https://docs.bigbluebutton.org/dev/webhooks.html
+You can read the full documentation at: http://docs.bigbluebutton.org/labs/webhooks.html
 
 
-Development
------------
+## Development
 
-1. Install node. You can use [NVM](https://github.com/creationix/nvm) if you need multiple versions of node or install it from source. To install from source, first check the exact version you need on `package.json` and replace the all `vX.X.X` by the correct version when running the commands below.
+With a [working development](https://docs.bigbluebutton.org/2.2/dev.html#setup-a-development-environment) environment, follow the commands below from within the `bigbluebutton/bbb-webhooks` directory.
 
-~~~
-wget http://nodejs.org/dist/vX.X.X/node-vX.X.X.tar.gz
-tar -xvf node-vX.X.X.tar.gz
-cd node-vX.X.X/
-./configure
-make
-sudo make install
-~~~
+1. Install the node dependencies:
+    - `npm install`
 
-2. Install the dependencies: `npm install`
+2. Copy the configuration file:
+    - `cp config/default.example.yml config/default.yml`
+    - Update the `serverDomain` and `sharedSecret` values to match your BBB server configuration in the newly copied `config/default.yml`.
 
-3. Copy and edit the configuration file: `cp config_local.coffee.example config_local.coffee`
+3. Stop the bbb-webhook service:
+    - `sudo service bbb-webhooks stop`
 
-4. Run the application with:
+4. Run the application:
+    - `node app.js`
+    - **Note:** If the `node app.js` script stops, it's likely an issue with the configuration, or the `bbb-webhooks` service may have not been terminated.
 
-~~~
-node app.js
-~~~
 
-5. To test it you can use the test application `post_catcher.js`. It starts a node app that
-  registers a hook on the webhooks app and prints all the events it receives. Open the file
-  at `extra/post_catcher.js` and edit the salt and domain/IP of your server and then run it
-  with `node extra/post_catcher.js`. Create meetings and do things on your BigBlueButton server
-  and the events should be shown in the post_catcher.
+### Persistent Hooks
 
-  Another option is to create hooks with the [API Mate](http://mconf.github.io/api-mate/) and
-  catch the callbacks with [PostCatcher](http://postcatcher.in/).
+If you want all webhooks from a BBB server to post to your 3rd party application/s, you can modify the configuration file to include `permanentURLs` and define one or more server urls which you would like the webhooks to post back to.
 
-Production
-----------
+To add these permanent urls, do the follow:
+ - `sudo nano config/default.yml`
+ - Add the configuration similar to this example:
+    - ```
+        permanentURLs: [
+            {
+                url: 'https://staging.example.com/webhook-post-route',
+                getRaw: false
+            },
+            {
+                url: 'https://app.example.com/webhook-post-route',
+                getRaw: false
+            }
+        ]
+      ```
```

Comment:
Two suggestions:
- permanentURLs is above the hooks attribute, so I suggest to modify the sentence to something like "Locate hooks.permanentURLs on config/default.yml and modify it as follows:"
- use the yaml syntax as follows:
```
permanentURLs:
  - url: 'https://staging.example.com/webhook-post-route'
    getRaw: false
  - url: 'https://app.example.com/webhook-post-route'
    getRaw: false
```